File: OOP Lab Final/lab1.java
import java.util.Scanner;

class Triangle {
    public double height;
    public double width;

    Triangle(double height, double width) {
        this.height = height;
        this.width = width;
    }

    double area() {
        return .5 * height * width;
    }

}

public class lab1 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        System.out.println("Enter the Height and width: ");
        double h = input.nextDouble();
        double w = input.nextDouble();
        Triangle T = new Triangle(h, w);
        double area = T.area();
        System.out.println("This area of the Triangle is: " + area);
        input.close();
    }
}

File: OOP Lab Final/lab_1.cpp
#include <bits/stdc++.h>
using namespace std;
class Triangle
{
public:
    double width, height;
    Triangle(double width, double height)
    {
        this->width = width;
        this->height = height;
    }
    double area()
    {
        return 0.5 * width * height;
    }
};
int main()
{
    double h, w, a;
    cout << "Enter the Height and Width  :" << endl;
    cin >> h >> w;
    Triangle object = Triangle(h, w);
    a = object.area();
    cout << "Triangle area is : " << a << endl;
    return 0;
}
